odata-protocol/11.4 Data Modification.md

```diff
@@ -1078,18 +1088,31 @@ If an individual change fails due to a failed dependency, it MUST be
 annotated with the term [`Core.DataModificationException`](https://github.com/oasis-tcs/odata-vocabularies/blob/main/vocabularies/Org.OData.Core.V1.md#DataModificationException) and SHOULD specify
 a `responseCode` of `424` ([Failed Dependency](#ResponseCode424FailedDependency)).
 
-Alternatively, the verb `PUT` can be used, in which case the request
-body MUST be the representation of a collection of entities. In this
+### ##subsubsec Replace a Collection of Entities
```

Comment:
Introducing a new subsection next to "Update a Collection of Entities" requires to repeat or reference everything from the preceding section that also applies here.

Reply:
That was my intention -- to include everything in this section that is relevant to replacing a collection of entities. Turns out, that's not much -- most of the section as written was really about updating the collection using a delta payload, which is why I thought it made sense to break replacing the collection into its own section.  What is relevant to replacing a collection is just:

1) Use PUT
2) Request is the entire collection, things omitted are considered removed
3) You can use ETags
4) If an error occurs, either apply no changes or return a representation showing the current membership.

Is there something else that was relevant to replacing a collection of entities in the original text that is missing from this new section?

Reply:
Use of `@etag` on individual entities is surprising. I'd treat this as a new feature.